integration/kubernetes/k8s-hugepages.bats

```diff
@@ -6,56 +6,64 @@
 
 load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
-issue="https://github.com/kata-containers/runtime/issues/2172"
 
 setup() {
-	skip "test not working see: ${issue}"
 	extract_kata_env
 
+	pod_name="hugepage-pod"
+	get_pod_config_dir
+
 	# Enable hugepages
-	sudo sed -i 's/#enable_hugepages = true/enable_hugepages = true/g' ${RUNTIME_CONFIG_PATH}
+	sed -i 's/#enable_hugepages = true/enable_hugepages = true/g' ${RUNTIME_CONFIG_PATH}
 
-	pod_name="test-env"
-	get_pod_config_dir
+	old_pages=`cat /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages`
+
+	# Set hugepage-2Mi to 4G(2Mi*2048)
+	echo 2048 > /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
```

Comment:
A couple of questions:

- Is it worth logging `$old_pages`?
- Minor point, but you could check the current value of `nr_hugepages` to see if it's different to `2048` before changing it.